src/p_interaction.cpp

```diff
@@ -145,15 +149,17 @@ void SexMessage (const char *from, char *to, int gender, const char *victim, con
 		}
 		else
 		{
-			int gendermsg = -1;
+			int type = -1;
```

Comment:
Like the other variable I think this should at least be labeled "pronountype" to be less ambiguous.

Reply:
Perhaps so, but verbosity is not a necessity in C code as far as I'm concerned. It's still obvious enough how it works.